bin/sl-deploy.js

```diff
@@ -62,8 +64,14 @@ var baseURL = argv[parser.optind()];
 if (numArgs === 2) {
   branchOrPack = argv[parser.optind() + 1];
 }
+baseURL = baseURL || 'http://';
 branchOrPack = branchOrPack || 'deploy';
 
+// Truncate any paths from the baseURL, because the git push does a raw string
+// concatenation of '/<config>', and older versions of deploy allowed paths on
+// the git push.
+baseURL = defaults(baseURL, {host: 'localhost', port: 8701}, {path: '/'});
```

Comment:
Might want to default to 127.0.0.1 instead of localhost. Depending on the OS, localhost may resolve to ::1 instead.